README.md

```diff
@@ -202,10 +202,14 @@ Before you start the exercises, please look at this [example](https://github.com
 
 In these exercises, you have a number of impure functions and a number of failing tests. Your task is to rewrite the functions (and not the tests!) to make sure that the tests pass and the functions are pure.
 
-* clone this repo and run `npm install`
-* run the tests using `npm test` (don't worry about the package-lock.json notice)
-* look at the passing and failing tests
-* open `exercises/exercise1.js`
-* refactor the functions to make the tests pass. It will be useful to open the tests and look at exactly what is expected.
+- clone this repo and run `npm install`
+- if you wish to use nodemon to automatically re-run your tests when you save:
+  - 1. run `npm i -D nodemon` in your terminal
+  - 2. change the test scripts in your package.json file to
```

Comment:
Would be useful to add a reference to scripts, it not mentioned anywhere else
https://medium.freecodecamp.org/introduction-to-npm-scripts-1dbb2ae01633

Reply:
Yeah true. Maybe we should mention npm scripts in the [Intro to npm](https://github.com/foundersandcoders/npm-introduction) they do at the start of the week

Reply:
@oliverjam yes could definitely be added to the intro, in the 4th task section

Reply:
Cool. I added [an issue](https://github.com/foundersandcoders/npm-introduction/issues/6) to that repo discussing it. Happy to resolve this if we're going to deal with this problem earlier in the week?

Reply:
Thank you :sparkles: